scripts/ios/create_ios_universal.sh

```diff
@@ -12,7 +12,7 @@ libtool -static -o ./target/CoEpiCore ./target/aarch64-apple-ios/release/libcoep
 
 # Overwrite library in iOS app (downloaded with Carthage) with local build.
 
-PATH_TO_IOS_REPO="<insert path>"
+PATH_TO_IOS_REPO="/Users/ischuetz/dev/covid/app-ios"
```

Comment:
We might need a better solution for PATH_TO_IOS_REPO

Reply:
Oops. Yeah, it should probably be an environment variable. Let's add a ticket for it.